spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -171,19 +161,19 @@ public void addDataUnpartitionedOrc() {
             "CALL %s.system.add_files('%s', '`orc`.`%s`')",
             catalogName, tableName, fileTableDir.getAbsolutePath());
 
-    Assert.assertEquals(2L, result);
+    Assertions.assertThat(result).isEqualTo(2L);
 
     assertEquals(
         "Iceberg table contains correct data",
         sql("SELECT * FROM %s ORDER BY id", sourceTableName),
         sql("SELECT * FROM %s ORDER BY id", tableName));
   }
 
-  @Test
+  @TestTemplate
   public void addAvroFile() throws Exception {
     // Spark Session Catalog cannot load metadata tables
     // with "The namespace in session catalog must have exactly one name part"
-    Assume.assumeFalse(catalogName.equals("spark_catalog"));
+    Assumptions.assumeFalse(catalogName.equals("spark_catalog"));
```

Comment:
please use the one from AssertJ: `assumeThat(catalog).isNotEqualTo("spark_catalog")`

Reply:
oops, sorry, let me fix this.